linked_lists/linked_list/linked_list.py

```diff
@@ -11,7 +11,7 @@ def __str__(self):
 class LinkedList(object):
 
     def __init__(self, head=None):
-        self.head = head
+        self.head = Node(None)
```

Comment:
I'm not sure why we are making this change?  What happens to the passed in `head` argument?